metacat-common/src/main/java/com/netflix/metacat/common/MetacatRequestContext.java

```diff
@@ -108,6 +108,22 @@ protected MetacatRequestContext(
         this.scheme = scheme;
     }
 
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("MetacatRequestContext{");
```

Comment:
Any particular reason you don't just use Jackson here and create a JSON object instead of all this manual serialization?

Reply:
Why would you want to create a JSON object and then do a toString.

Reply:
I don't know the use of your toString here but I'm saying return a JSON string from the toString.

Reply:
It is a toString method that prints out the instance variable values. Today we log this for every API call. It is not in JSON format. I auto-generated this method in intellij.